server.js

```diff
@@ -1,23 +1,99 @@
 const express = require("express");
 const cors = require("cors");
+const bodyParser = require("body-parser");
 
 const app = express();
 
-app.use(cors());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+const port = 3007;
+
+app.use(
+  cors({
+    origin: "http://localhost:3007",
+  })
+);
 
 const welcomeMessage = {
   id: 0,
-  from: "Bart",
+  from: "Pakize",
   text: "Welcome to CYF chat system!",
+  timeSpent: "24 / 12 / 2022"
 };
 
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
 const messages = [welcomeMessage];
 
-app.get("/", function (request, response) {
-  response.sendFile(__dirname + "/index.html");
+
+//Read only text whose text contains a given substring
+
+app.get("/messages/search", (req, res) => {
+  const { term } = req.query;
+  console.log(term);
```

Comment:
It's good practice to clean up your debug statements before raising a PR

Reply:
:+1: